arwen/context/arwen.go

```diff
@@ -34,6 +43,7 @@ type vmContext struct {
 	blockChainHook vmcommon.BlockchainHook
 	cryptoHook     vmcommon.CryptoHook
 	imports        *wasmer.Imports
+	instance       wasmer.Instance
```

Comment:
Is this intended as a step towards reusing a Wasmer instance?

Reply:
yes.